README.md

```diff
@@ -52,6 +52,32 @@ any suggestions! All of the software here is written by students.
 10. Point your browser to [`http://localhost:3000`](http://localhost:3000)
     :heart_eyes:
 
+##### Windows Alternative w/ WSL Ubuntu
```

Comment:
This is completely optional and up to you (@starruby15) since you normally use Windows for development, but I think this is a superior method of setting up the website since a lot of things rely on a Bash shell. Maybe move this up to make it the main method in README.md and re-label the other method as the "alternative" instead?